webapp/src/view/home/components/AppCard.tsx

```diff
@@ -77,7 +74,12 @@ export default function AppCard({
   };
 
   const handleLaunchClick = () => {
-    window.open(appUrl, "_blank", "noopener,noreferrer");
+    if (!appUrl) return; 
+        const url = appUrl.startsWith('http://') || appUrl.startsWith('https://') 
```

Comment:
Move `'http://'` and `'https://' to constants


```suggestion
        const url = appUrl.startsWith(HTTP_PREFIX) || appUrl.startsWith(HTTPS_PREFIX) 
```

Reply:
Do we really need to make a constant for `http://` and `https://` ? Do we have any reasons?